src/docs/asciidoc/api.adoc

```diff
@@ -464,7 +464,12 @@ outstanding unconfirmed messages timed out.
 |Time before enqueueing of a message fail when the maximum number of unconfirmed
 is reached. The callback of the message will be called with a negative status.
 Set the value to `Duration.ZERO` if there should be no timeout.
-|10 seconds.
+|10 seconds
+
+|`retryOnRecovery`
+|Re-publish unconfirmed messages when restoring a connection.
```

Comment:
The term we use in the docs is "connection recovery", so, "after recovering a connection"

Reply:
Thanks